### PR TITLE
fix: Update counter Thing

### PR DESCRIPTION
**Pull request by egekorkan**
There were inconsistencies with the multiple language options. This was actually resulting in an invalid TD. I have also adjusted some description text and google translated it to other languages.

### CODE REVIEW

**Review by danielpeintner — APPROVED** [review]
Thanks, LGTM! 

I wonder why the original source has the _required_ quotes?
https://github.com/eclipse/thingweb.node-wot/blob/ffd208636e288670aaa75dc5c394ee10a782b1c7/packages/examples/src/scripts/counter.ts#L67

Anyhow, the PR looks good 👍

**Comment by danielpeintner** [comment]
@egekorkan It seems you ought to run 
`npm run lint`
and/or
`npm run format`

**Review by danielpeintner — CHANGES_REQUESTED** [review]
@egekorkan It seems you ought to run 
`npm run lint`
and/or
`npm run format`

BTW, when first approving I just looked at the missing quotes.
Give me more time to check the other changes..  which need to be reflected in https://github.com/eclipse/thingweb.node-wot/blob/master/packages/examples/src/scripts/counter.ts also

File: examples/scripts/counter.js [hunk 1]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
+                de: "Verringern",
+                it: "Decrementa"
+            },
             description: "Decrementing counter value",
             descriptions: {
                 en: "Decrementing counter value",
-                de: "Zähler verringern",
-                it: "decrementare valore",
-            },
+                de: "Zählerwer verringern",
```
Comment (danielpeintner) [inline examples/scripts/counter.js:154]: t missing

File: examples/scripts/counter.js [hunk 2]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
+            minimum: 1,
+            maximum: 250
+        }
     },
     properties: {
         count: {
+            title: "Count",
+            titles: {
+                en: "Count",
+                de: "Zählen",
```
Comment (danielpeintner) [inline examples/scripts/counter.js:67]: Zähler?

File: examples/scripts/counter.js [hunk 3]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
+                it: "Conteggio"
```
Comment (danielpeintner) [inline examples/scripts/counter.js:68]: Valore ?

File: examples/scripts/counter.js [hunk 4]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
+            },
             type: "integer",
-            description: "current counter value",
+            description: "Current counter value",
             descriptions: {
-                en: "current counter value",
+                en: "Current counter value",
                 de: "Derzeitiger Zähler Stand",
```
Comment (danielpeintner) [inline examples/scripts/counter.js:74]: At the bottom we name it Zählerwert. We should be consistent.

File: examples/scripts/counter.js [hunk 5]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
-            description: "current counter value as SVG image",
-            forms: [
-                {
-                    contentType: "image/svg+xml",
-                },
-            ],
+            description: "Current counter value as SVG image",
+            descriptions: {
+                en: "Current counter value as SVG image",
+                de: "Aktueller Zählerstand als SVG-Bild",
```
Comment (danielpeintner) [inline examples/scripts/counter.js:85]: `Zählerstand` vs. `Zähler Stand`  above

File: examples/scripts/counter.js [hunk 6]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
             uriVariables: {
-                fill: { type: "string" },
+                fill: {
+                    type: "string"
+                }
             },
-        },
-        redDotImage: {
-            description: "Red dot image as PNG",
             forms: [
```
Comment (danielpeintner) [inline examples/scripts/counter.js:95]: why has the form with contentType: "image/png" be removed ?

File: examples/scripts/counter.js [hunk 7]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
         increment: {
-            description: "Incrementing counter value",
-            descriptions: {
-                en: "Incrementing counter value",
-                de: "Zähler erhöhen",
-                it: "incrementare valore",
+            title: "Increment",
+            titles: {
+                en: "Increment",
+                de: "Zunahme",
```
Comment (danielpeintner) [inline examples/scripts/counter.js:134]: Erhöhen oder Inkrementieren ?

File: examples/scripts/counter.js [hunk 8]
```diff
@@ -48,95 +48,144 @@ WoT.produce({
...
+                de: "Zählerwer verringern",
+                it: "Decrementare il valore del contatore"
+            }
         },
         reset: {
+            title: "Reset",
+            titles: {
+                en: "Reset",
+                de: "Zurücksetzen",
+                it: "Ripristina"
```
Comment (danielpeintner) [inline examples/scripts/counter.js:163]: Reset ?

**Review by danielpeintner — CHANGES_REQUESTED** [review]
Some more things I stumbled over...

File: examples/scripts/counter.js [hunk 9]
```diff
@@ -86,54 +105,89 @@ WoT.produce({
...
+                de: "Letzte Änderung",
+                it: "Ultima modifica",
+            },
             type: "string",
-            description: "last change of counter value",
+            description: "Last change of counter value",
             descriptions: {
-                en: "last change of counter value",
+                en: "Last change of counter value",
                 de: "Letzte Änderung",
```
Comment (JKRhb) [inline examples/scripts/counter.js:127]:
```suggestion
                de: "Letzte Zählerwertänderung",
```

File: examples/scripts/counter.js (hunk 1 above)
Comment (JKRhb) [reply examples/scripts/counter.js:159]:
```suggestion
                de: "Zählerwert verringern",
```

File: examples/scripts/counter.js [hunk 10]
```diff
@@ -86,54 +105,89 @@ WoT.produce({
...
+                en: "Reset",
+                de: "Zurücksetzen",
+                it: "Reset",
+            },
             description: "Resetting counter value",
             descriptions: {
                 en: "Resetting counter value",
-                de: "Zähler resettieren",
-                it: "resettare valore",
+                de: "Zählerwert resettieren",
```
Comment (JKRhb) [inline examples/scripts/counter.js:173]: I think "zurücksetzen" should be a more common translation.

```suggestion
                de: "Zählerwert zurücksetzen",
```

File: examples/scripts/counter.js [hunk 11]
```diff
@@ -86,54 +105,89 @@ WoT.produce({
...
+            titles: {
+                en: "Changed",
+                de: "Geändert",
+                it: "Valore modificato",
+            },
+            description: "Change event",
             descriptions: {
-                en: "change event",
+                en: "Change event",
                 de: "Änderungsnachricht",
```
Comment (JKRhb) [inline examples/scripts/counter.js:189]: "-nachricht" could be replaced with "-ereignis" to achieve a more literal translation, the current version should also be fine :)

**Review by JKRhb — COMMENTED** [review]
See a couple of minor suggestions regarding the German translation below :)

**Comment by egekorkan** [comment]
@danielpeintner @JKRhb I have fixed the wording and ran linting

File: examples/scripts/counter.js [hunk 12]
```diff
@@ -25,19 +25,19 @@
...
-    description: "counter example Thing",
+    description: "Counter example Thing",
     descriptions: {
-        en: "counter example Thing",
+        en: "Counter example Thing",
         de: "Zähler Beispiel Ding",
         it: "Contatore Esempio",
     },
-    support: "https://github.com/eclipse/thingweb.node-wot/",
+    support: "git://github.com/eclipse/thingweb.node-wot.git",
```
Comment (danielpeintner) [inline examples/scripts/counter.js:40]: Why do you want to change the support link which can be viewed in a browser to a git link that does **not** render in a browser and does not give any support?

**Review by danielpeintner — CHANGES_REQUESTED** [review]
LGTM besides the following things

* comment below about `support`
* this file is auto-generated by https://github.com/eclipse/thingweb.node-wot/blob/master/packages/examples/src/scripts/counter.ts. We need to align the source as well. Copying the TD over and run linting on both files again should do the job..
* _another picky note which was there before. the `forms  entry `in `countAsImage `comes last while in `redDotImage `it comes before. Not sure if it is worth to align the sorting of some entries. Technically it does not matter and we can leave it as is...

File: examples/scripts/counter.js (hunk 12 above)
Comment (egekorkan) [reply examples/scripts/counter.js:40]: The one submitted to wot-testing had this, probably due to being older than the master branch here. I will fix it

**Comment by egekorkan** [comment]
@danielpeintner I have incorporated your feedback

File: packages/examples/src/scripts/counter.ts [hunk 13]
```diff
@@ -27,15 +27,15 @@ let count: number;
...
+        de: "Zähler",
         it: "Contatore",
     },
-    description: "counter example Thing",
+    description: "Counter example Thing",
     descriptions: {
-        en: "counter example Thing",
+        en: "Counter example Thing",
         de: "Zähler Beispiel Ding",
         it: "Contatore Esempio",
```
Comment (relu91) [inline packages/examples/src/scripts/counter.ts:40]:
```suggestion
        it: "Contatore di esempio",
```

**Review by relu91 — APPROVED** [review]
Minor thing left

**Comment by relu91** [comment]
@danielpeintner good to go?

**Comment by egekorkan** [comment]
I have incorporated the feedback and resolved the conversations above. It should be good to go now :)

**Review by danielpeintner — APPROVED** [review]
Thanks @egekorkan

**Pull request merged**